### PR TITLE
RDoc-1963 Locate & add missing Java articles for RavenDB 5.2

### PR DESCRIPTION
**Pull request by Drorlev**

### CODE REVIEW

File: Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/create-client-certificate.java.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **name** | String | Name of a certificate |
+| **permissions** | Map&lt;String, DatabaseAccess&gt; | Map with database to access level mapping |
+| **clearance** | SecurityClearance | Access level |
+| **password** | String | Optional certificate password, default: no password |
+
+| Return Value | |
+| ------------- | ----- |
+| **RawData** | client certificate raw data |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/create-client-certificate.java.markdown:22]: this is the most trick field - it isn't certificate raw data but zip which contains few certs... IIRC

Can you check that?

File: Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/put-client-certificate.java.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+{CODE:java cert_1_2@ClientApi\Operations\Server\ClientCertificate.java /}
+
+{CODE:java cert_1_3@ClientApi\Operations\Server\ClientCertificate.java /}
+
+
+| Parameters | | |
+| ------------- | ------------- | ----- |
+| **name** | String | Name of a certificate |
+| **certificate** | string | Certificate to register |
```
Comment (ml054) [inline Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/put-client-certificate.java.markdown:17]: string -> String

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            CLUSTER_ADMIN,
+            CLUSTER_NODE,
+            OPERATOR,
+            VALID_USER
+        }
+        //endregion
+
+        //region cert_1_3
+        public enum DatabaseAccess {
+            READ_WRITE,
```
Comment (ml054) [inline Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java:65]: Don't we have READ here as well starting from 5.2?

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    public ClientCertificate() {
+        try (IDocumentStore store = new DocumentStore()) {
+            //region cert_1_4
+            // With user role set to Cluster Administrator or Operator the user of this certificate
+            // is going to have access to all databases
+
+            CreateClientCertificateOperation operation = new CreateClientCertificateOperation("admin",
+                null, SecurityClearance.OPERATOR);
+            CertificateRawData certificateRawData = store.maintenance().server().send(operation);
+            byte[] certificatesZipped = certificateRawData.getRawData();
```
Comment (ml054) [inline Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java:81]: here you have it - raw data is a zip :)

File: Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/put-client-certificate.java.markdown (hunk 2 above)
Comment (Drorlev) [reply Documentation/5.2/Raven.Documentation.Pages/client-api/operations/server-wide/certificates/put-client-certificate.java.markdown:17]: Done

File: Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java (hunk 3 above)
Comment (Drorlev) [reply Documentation/5.2/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Server/ClientCertificate.java:66]: Done

**Review by ml054 — APPROVED** [review]

**Pull request merged**